app/Database/Schema/AccountSchema.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Database\Schema;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @method null|string getAccountId()
+ * @method null|string getAccountNumber()
+ * @method null|string getSubscriptionType()
+ * @method null|string getUser()
+ * @method self setAccountNumber(string $accountNumber)
+ * @method self setSubscriptionType(string $subscriptionType)
+ * @method self setUser(\App\Database\Schema\User $user)
+ */
+trait AccountSchema
+{
+    /**
+     * @ORM\Column(type="guid", name="id")
+     * @ORM\GeneratedValue(strategy="UUID")
+     * @ORM\Id()
+     *
+     * @var string
+     */
+    protected $accountId;
+
+    /**
+     * @ORM\Column(type="string", name="account_number")
+     *
+     * @var string
+     */
+    protected $accountNumber;
+
+    /**
+     * @ORM\Column(type="string", name="subscription_type")
+     *
+     * @var string
+     */
+    protected $subscriptionType;
+
+    /**
+     * @ORM\ManyToOne(targetEntity="User", inversedBy="account")
+     * @ORM\JoinColumn(nullable=false)
+     * 
+     * @var \App\Database\Schema\User
+     */
+    protected $user;
```

Comment:
As part of our convention and readability, we only define database columns in schema traits. While this would work, let's move relationship annotations into the entity class.

Another one is we need to define the foreign key column manually rather than having doctrine(magic) to do it. So in the schema please add `user_id` column with guid type and is nullable. ;p